examples/image-builder.yaml

```diff
@@ -49,36 +51,52 @@ deployment_groups:
     source: modules/packer/custom-image
     kind: packer
     settings:
-      disk_size: 20
       source_image_project_id: [schedmd-slurm-public]
-      source_image_family: schedmd-slurm-21-08-8-hpc-centos-7
+      # see latest in https://github.com/SchedMD/slurm-gcp/blob/master/docs/images.md#supported-operating-systems
+      source_image_family: schedmd-v5-slurm-22-05-6-hpc-centos-7
       image_family: $(vars.new_image_family)
+      state_timeout: 15m
```

Comment:
Is it needed here? (if you decrease the image size)

Reply:
One of my tries failed with packer timeout.

Reply:
Sorry, my comment wasn't clear. This is likely needed because the image is 100GB, but if that is reduced, then you might not need this.